### DISCUSSION TITLE
Scripts duplicated at runtime in header and the body

### DISCUSSION THREAD

**Discussion by raulgcode** [nuxt/nuxt#19028] [Questions (Nuxt 2)]
I am facing an odd issue where some bundle scripts are being added twice, besides the preload link. Also this behavior is not consistent to get the duplication I need to reload several times. For the business is a problem since it increases the page size and it impacts the SEO and how the crawlers see the page. Here is some screenshots for what I am facing
As you can see `31.de772fc.modern.js` was added twice, one script tag in the header and the other in the body. 

<img width="618" alt="image" src="https://user-images.githubusercontent.com/4565524/218776188-49013843-236a-4957-ba4c-3f237800b58c.png">

Do you know what is happening here? And how to solve it?
Thanks in advance

**Comment by TheAlexLichter** [comment]
Rather unsure why. Could you provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
